test/kvao/delete-all-fallback.suite.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const bdd = require('../helpers/bdd-if');
+const DataSource = require('../..').DataSource;
+const helpers = require('./_helpers');
+const kvMemory = require('../../lib/connectors/kv-memory');
+const should = require('should');
+
+module.exports = function(dataSourceFactory, connectorCapabilities) {
+  // override injected data source factory
```

Comment:
See my comment above, the decision fallback vs regular deleteAll is made by the connector. There is no point to test how connectors implementing optimized deleteAll are handling the fall-back variant, because the fall-back variant will be never invoked.

This whole file should be deleted and [test/kv-memory.test.js](https://github.com/strongloop/loopback-datasource-juggler/blob/66c54a96467b39609fec07f41b6d8018c5fc3819/test/kv-memory.js#L5-L11) should be modified to run the shared test suite twice.